Domain/FolderEntity.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Domain;
+
+public class FolderEntity
+{
+    public int Id { get; set; }
```

Comment:
Please make entity with id and inherit other from it. ID should be[ UUID ](https://stackoverflow.com/questions/23081096/entity-framework-6-guid-as-primary-key-cannot-insert-the-value-null-into-column)